packages/react-instantsearch-core/src/widgets/QueryRuleContext.ts

```diff
@@ -0,0 +1,3 @@
+import connectQueryRules from '../connectors/connectQueryRules';
+
+export default connectQueryRules(() => null);
```

Comment:
This function is easier to read in the dev tools like this: 

```suggestion
export default connectQueryRules(function QueryRuleContext() {
  return null;
});
```

I changed it to that form for Configure too in #2179